File: php/entity/person.php
<?php
	class Person{
		private $id;	
		private $firstName;
		private $lastName;
		private $email;
		
		public function setId($id){
			$this->id = $id;
		}

		public function setFirstName($firstName){
			$this->firstName = $firstName;
		}

		public function setLastName($lastName){
			$this->lastName = $lastName;
		}
		public function setEmail($email){
			$this->email = $email;
		}

		public function getId(){
			return $this->id;
		}

		public function getFirstName(){
			return $this->firstName;
		}

		public function getLastName(){
			return $this->lastName;
		}
		
		public function getEmail(){
			return $this->email;
		}
	}
?>

File: python/entity/person.py
class Person:
    __id = None
    __firstName = None
    __lastName = None
    __email = None

    def setId(self,id):
        self.__id = id

    def setFirstName(self,firstName):
        self.__firstName = firstName

    def setLastName(self,lastName):
        self.__lastName = lastName

    def setEmail(self,email):
        self.__email = email
    
    def getId(self):
        return self.__id
    
    def getFirstName(self):
        return self.__firstName

    def getLastName(self):
        return self.__lastName

    def getEmail(self):
        return self.__email

